### DISCUSSION TITLE
having trouble with alt account

### DISCUSSION THREAD

**Discussion by esp8266nodemcu860** [revoxhere/duino-coin#1762] [Q&A]
i opened multiple accounts to test the miner effeciency then i ended with this error massage "You're using multiple accounts: password vacuum_cleaner notalegend Herlambang7 Espminer ammorack avrminer ourammorack superrainbow supermario" any way to remove those accounts?

**Comment by Jonny11bonk** [comment]
Opening multiple accounts in the first place is your problem and by that, you've violated the Terms of Service you've agreed to upon a registration.
